rpm/harbour-contrac.spec

```diff
@@ -83,10 +83,10 @@ fi
 %{_datadir}/icons/hicolor/*/apps/%{name}.png
 %{_datadir}/%{name}/translations
 %{_bindir}/contracd
-%{_libdir}/systemd/user/contracd.service
+%{_userunitdir}/contracd.service
```

Comment:
Doesn't this also need a change here? https://github.com/llewelld/harbour-contrac/blob/25b36142001bb1880f6bd71f8c6734f1ca1f0836/contracd/contracd.pro#L92

Reply:
Nevermind it's fine, I overlooked the `/user/` and got confused.